table_transforms/market_basket/market_basket.yaml

```diff
@@ -0,0 +1,22 @@
+name: market_basket
+description: |
+  Analyze historical transaction contents to understand products that are frequently purchased together.
+
+  This approach uses a transactional table to aggregate each product purchased in a transaction, and then aggregates transactions together to look for common patterns.
+arguments:
+  transaction_id:
+    type: column
+    description: Column to group multiple line items into a single transaction
+  sep:
+    type: value
+    description: Text separator to use when aggregating the strings, i.e. ', ' or '|'.
+  agg_column:
+    type: column
+    description: Product ID or description to use when aggregating into transactions
```

Comment:
I think this var name & description suffers from the same complaint above. My personal opinion, I'm happy to hear counter arguments